### DISCUSSION TITLE
How do Hostage entity ids work?

### DISCUSSION THREAD

**Discussion by Rebze** [akiver/cs-demo-manager#838] [Q&A]
Does anyone know how the Hostage entity ids work?
While going through the database I found the tables public.hostage_picked_up and public.hostage_rescued empty. The tables public.hostage_pick_up_start and public.hostage_rescued are filled, but some hostage entity ids in public.hostage_rescued do not have corresponding entries in hostage_pick_up_start. This happened for about half the successful rescues.

I have tried around with a few demos, both Valve and FACEIT, but haven't been able to find out when/why it happens - sometimes it even happens with hostages that were only picked up once and carried straight to the rescue zone without any interruptions. I can provide an example demo if necessary, it's just too big to upload here

**Comment by akiver** [comment]
Hi!

> The tables public.hostage_pick_up_start and public.hostage_rescued are filled, but some hostage entity ids in public.hostage_rescued do not have corresponding entries in hostage_pick_up_start. 

This will be fixed in the next release.
